tests/PKSim.Tests/Core/WorkspaceSpecs.cs

```diff
@@ -250,7 +250,7 @@ protected override void Because()
       [Observation]
       public void should_update_the_path_to_a_relative_path()
       {
-         sut.Project.JournalPath.ShouldBeEqualTo("..\\journal.sbj");
+         sut.Project.JournalPath.ShouldBeEqualTo("journal.sbj");
```

Comment:
Correct implementation in core again. The JournalPath is relative to project folder